runtime/debug/block.go

```diff
@@ -0,0 +1,30 @@
+package debug
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/prysmaticlabs/prysm/v5/config/features"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
+	"github.com/prysmaticlabs/prysm/v5/io/file"
+	log "github.com/sirupsen/logrus"
+)
+
+// SaveBlockToTemp saves block to temp directory in ssz format.
+func SaveBlockToTemp(block interfaces.ReadOnlySignedBeaconBlock) {
+	if !features.Get().SaveInvalidBlock && !features.Get().SaveUnavailableBlock {
```

Comment:
Are you sure this is the boolean logic you want? Only save to temp when both flags are present?

Reply:
oh wait nvm it only returns when both flags are false. Harder to read the negative.